jaxrs-clients/src/main/java/com/palantir/remoting2/jaxrs/JaxRsClient.java

```diff
@@ -55,6 +63,28 @@ private JaxRsClient() {}
                 serviceConfiguration -> create(serviceClass, userAgent, serviceConfiguration)));
     }
 
+    public static javax.ws.rs.client.ClientBuilder jaxrsBuilder(ServiceConfiguration serviceConfig) {
+        ClientConfig config = ClientConfig.fromServiceConfig(serviceConfig);
+        JerseyClientBuilder clientBuilder = new JerseyClientBuilder();
+        clientBuilder.register(new JacksonJaxbJsonProvider(ObjectMappers.newClientObjectMapper(), JacksonJaxbJsonProvider.BASIC_ANNOTATIONS));
+        clientBuilder.property(CommonProperties.FEATURE_AUTO_DISCOVERY_DISABLE, Boolean.TRUE);
+
+        if (serviceConfig.security().isPresent()) {
+            clientBuilder.sslContext(SslSocketFactories.createSslContext(serviceConfig.security().get()));
+        }
+        clientBuilder.property(ClientProperties.CONNECT_TIMEOUT, config.connectTimeout().toMilliseconds());
+        clientBuilder.property(ClientProperties.READ_TIMEOUT, config.writeTimeout().toMilliseconds());
+        if (config.proxy().isPresent() && config.proxy().get().type() == Type.HTTP) {
```

Comment:
I found some stack overflow code about making a HttpURLConnectionFactory but I haven't tried it